std/concurrency.d

```diff
@@ -1716,15 +1716,7 @@ void yield(T)(T value)
 }
 
 
-version (Win64)
-{
-    // fibers are broken on Win64
-}
-else version (Win32)
-{
-    // fibers are broken in Win32 under server 2012: bug 13821
-}
-else unittest
+unittest
 {
```

Comment:
> I'm not sure if Brad moved the tester to 2012

From the threads it's not clear whether @braddr managed to make the move:

https://github.com/dlang/druntime/pull/150 
http://forum.dlang.org/post/5480C893.4080507@puremagic.com

> I know this is ancient code, but it's failing on windows server 2012 for 32 bit windows apps.